session/session_test.go

```diff
@@ -21,23 +21,28 @@ func TestVerifySign(t *testing.T) {
 	cookie, _ := url.QueryUnescape(signedCookie)
 	vectors := strings.SplitN(cookie, "--", 2)
 	// a right signature case
-	if verified := verifySign(vectors[0], vectors[1], secretKeyBase, signSalt); !verified {
-		t.Error("The function verifySign() test failure!")
+	verified, err := verifySign(vectors[0], vectors[1], secretKeyBase, signSalt)
```

Comment:
Same here, not sure if not handling errors is a good idea.

Reply:
same as above.